src/GenomeCopyNumber.cpp

```diff
@@ -1171,20 +1174,29 @@ long double GenomeCopyNumber::calculateRSS(int ploidy)
 	}
 
     long double RSS = 0;
+    auto out_file_name = "RSS_expected_observed.p" + std::to_string(ploidy) + ".tsv";
```

Comment:
this print shows us the effect of correction for ploidy for each ploidy value. 
if we aim at trying ploidy estimation again this can be useful. 
removing for now.